packages/tools/test/apm.coffee

```diff
@@ -0,0 +1,63 @@
+nikita = require '@nikitajs/core'
+{tags, ssh} = require './test'
+they = require('ssh2-they').configure ssh...
+
+return unless tags.tools_apm
+
+describe 'tools.apm', ->
+
+  they 'apm is installed on system', ({ssh}) ->
+    nikita
+      ssh: ssh
+    .system.execute
+      cmd: """
+            if
+            apm -v | grep apm
+            then
+            exit 0
+            else
+            exit 1
+            fi
+            """
+    .promise()
+
+
+  they 'install a new package which is not already installed', ({ssh}) ->
+    nikita
+      ssh: ssh
+    .system.execute
+      cmd: """
+            if
+            apm list | grep package-list
+            then
+            apm uninstall package-list
+            fi
```

Comment:
please condense your code, this creates useless scrolling without readable benefits.

Reply:
OK. Bringing in to single liners.